### DISCUSSION TITLE
How to add multiple folders in exclude_dir?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [florianschanda/miss_hit#274] [Q&A] [answered]
The document here https://florianschanda.github.io/miss_hit/style_checker.html

does not mention how to exclude multiple directories? It would be great to clarify, thanks! ☺️

**Comment by florianschanda** [comment, marked as answer]
You can just use the same directive twice, so:
```
exclude_dir: "a"
exclude_dir: "b"
```

Remember the MH config files are basically scripts, they are executed top-to-bottom. For some config options it just overwrites the previous value, for some it adds.

**Reply by hongbo-miao** [reply]
I see, thanks for clarification!
